documentation/single-file-tools.md

```diff
@@ -0,0 +1,66 @@
+# Single File Diagnostic Tools
+
+All of our diagnostic tools have been distributed so far as [global tools](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools). These provide a great user experience as they provide clear ways to easily download, update, and configure the scope of the tools. However, these require a full .NET SDK which is often not available in some scenarios like production machines. For these scenarios we provide a single file distribution mechanism that only require a runtime to be available in the target machine. Other than the installation instructions, all functionality and usage remains the same. If you need help on how to use them, please refer to the [.NET Core diagnostic global tools](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/#net-core-diagnostic-global-tools) documentation.
+
+## Requirements to Run the Tools
+
+- The tools distributed in a single file format are framework dependent applications. As such, they require a 3.1+ .NET Core runtime to be available. This means one of the following:
```

Comment:
I would add something about customers using container images where the runtime is already installed globally. This is probably a common scenario where customers will want to acquire these tools.